routing/routing_integration_tests/routing_test_tools.cpp

```diff
@@ -216,6 +218,15 @@ namespace integration
         ("Route time test failed. Expected:", expectedRouteSeconds, "have:", routeSeconds, "delta:", delta));
   }
 
+  void TestRoutePointsNumber(Route const & route, size_t expectedPointsNumber, double relativeError)
+  {
+    CHECK_GREATER_OR_EQUAL(relativeError, 0.0, ());
+    size_t const routePoints = route.GetPoly().GetSize();
+    TEST(my::AlmostEqualRel((double)routePoints, (double)expectedPointsNumber, relativeError),
```

Comment:
We generally use static_cast instead of this old C-cast. Please, fix this.

Reply:
done